runtime/src/xcm_config.rs

```diff
@@ -50,7 +46,7 @@ use crate::weights::pallet_xcm::ZKVWeight as XcmPalletZKVWeight;
 use crate::weights::xcm::ZKVWeight as XcmZKVWeight;
 
 const ZKV_GENESIS_HASH: [u8; 32] =
-    hex_literal::hex!("e2a4f521dbcba897cd2359adc5e7725f409b17f9ae129737904e5e8939c22a05");
+    hex_literal::hex!("ff7fe5a610f15fe7a0c52f94f86313fb7db7d3786e7f8acf2b66c11d5be7c242");
```

Comment:
This is the current genesis hash right ? The old one was outdated ?

Reply:
Yes, this is Volta's. The previous genesis referred to the pre-regenesis chain.